pkgs/applications/misc/unixcw/default.nix

```diff
@@ -0,0 +1,36 @@
+{stdenv, fetchurl, libpulseaudio, alsaLib , pkgconfig, qt5}:
+stdenv.mkDerivation rec {
+  name = "unixcw-${version}";
+  version = "3.5.1";
+  src = fetchurl {
+    url = mirror://sourceforge/unixcw/unixcw_3.5.1.orig.tar.gz;
+    sha256 ="5f3aacd8a26e16e6eff437c7ae1e9b389956fb137eeb3de24670ce05de479e7a";
+  };
+  patches = [
+    ./remove-use-of-dlopen.patch.gz
```

Comment:
gzipping this patch seems excessive to me; how about we just have the .patch file unzipped.